src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -230,8 +231,8 @@ function MoneyRequestView({report, betas, parentReport, policyCategories, should
                     <MenuItemWithTopDescription
                         description={translate('common.date')}
                         title={transactionDate}
-                        interactive={canEdit}
-                        shouldShowRightIcon={canEdit}
+                        interactive={canEdit && !isSettled}
```

Comment:
Curious why we didn't use here `canEditFieldOfMoneyRequest` and pass the `date` field to it?

Reply:
Looks like we actually can use `canEditFieldOfMoneyRequest` for those 3 specific fields of `MoneyRequestView`. 
@youssef-lr, can I create a follow up PR for the issue - https://github.com/Expensify/App/issues/28963?

Reply:
It's okay, I'm working on this [issue](https://github.com/Expensify/App/issues/29914) and will include this update.

Reply:
Coo, thank you!